tutorials/Python/neonUtilities/neonUtilitiesPython.md

```diff
@@ -0,0 +1,342 @@
+
+Before starting, you will need:
+1. Python 3 installed. It is probably possible to use this workflow in Python 2, but these instructions were developed and tested using 3.6.4.
+2. R installed. You don't need to have ever used it directly. We tested using R 3.4.3, but most other recent versions should also work.
+3. rpy2 installed. Run the line below from the command line. It won't run within Jupyter, the error message is shown for clarity. It also likely will not work on Windows, see next section if you are running Windows.
+
+
+```python
+pip install rpy2
+```
+
+    
+    The following command must be run outside of the IPython shell:
+    
+        $ pip install rpy2
+    
+    The Python package manager (pip) can only be used from outside of IPython.
+    Please reissue the `pip` command in a separate terminal or command prompt.
+    
+    See the Python documentation for more informations on how to install packages:
+    
+        https://docs.python.org/3/installing/
+
+
+### Windows users
+
+The rpy2 package was built for Mac, and doesn't always work smoothly on Windows. If you have trouble with the install, try these steps.
+
+1. Add C:\Program Files\R\R-3.3.1\bin\x64 to the Windows Environment Variable “Path”
+2. Install rpy2 manually from https://www.lfd.uci.edu/~gohlke/pythonlibs/#rpy2
+    1. Pick the correct version. At the download page the portion of the files with cp## relate to the Python version. e.g., rpy2 2.9.2 cp36 cp36m win_amd64.whl is the correct download when running Python 36 and 64 bit Windows (amd64).
+    2. Save the whl file, navigate to it in windows then run pip directly on the file as follows “pip install rpy2 2.9.2 cp36 cp36m win_amd64.whl”
+3. Add  an R_HOME Windows environment variable with the path C:\Program Files\R\R-3.4.3 (or whichever version you are running)
+4. Add an R_USER Windows environment variable with the path C:\Users\tgoulden\AppData\Local\Continuum\Anaconda3\Lib\site-packages\rpy2 (modify for your own user account)
+
+## Load packages
+
+Now import rpy2 into your session.
+
+
+```python
+import rpy2
+import rpy2.robjects as robjects
+from rpy2.robjects.packages import importr
+```
+
+Import the base R functionality.
+
+
+```python
+base = importr('base')
+utils = importr('utils')
+```
+
+Suppress R warnings. This step can be skipped, but will result in messages getting passed through from R that Python will interpret as warnings.
```

Comment:
What is the reason for having these instructions here instead up with with the other imports?

Reply:
I don't think I understand - this is in the setup section, along with the importing. ?